src/test/java/com/celements/model/classes/fields/NumberFieldTest.java

```diff
@@ -29,8 +29,7 @@
   @Before
   public void setUp() throws Exception {
     super.setUp();
```

Comment:
please do not override the default setUp method. Instead add a different named method with Before annotation.